src/index.html

```diff
@@ -2,11 +2,392 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="#">
```

Comment:
Check code style rules [HERE](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-formatting), plz, fix in all cases
```suggestion
    <link 
      rel="preconnect"
      href="https://fonts.gstatic.com"
      crossorigin="#"
    >
```